app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -26,6 +31,25 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
     miqService.sparkleOff();
   }
 
+  $scope.storageManagerChanged = function (id) {
+    miqService.sparkleOn();
+    
+    $http.get('/cloud_volume/storage_manager_form_fields/' + id)
+      .then(getStorageManagerFormDataComplete)
+      .catch(miqService.handleFailure);
+  }
+
+  function getStorageManagerFormDataComplete(response) {
+    var data = response.data;
+
+    $scope.afterGet = true;
+    $scope.cloudVolumeModel.emstype = data.emstype;
```

Comment:
`cloudVolumeModel` should constitute all the form data.

`$scope.cloudVolumeModel.emstype = data.emstype;` looks correct.